src/software/ai/hl/stp/tactic/passer/simulated_passer_tactic_test.cpp

```diff
@@ -0,0 +1,127 @@
+#include <gtest/gtest.h>
+
+#include <utility>
+
+#include "software/ai/hl/stp/tactic/move/move_tactic.h"
+#include "software/ai/hl/stp/tactic/passer/passer_tactic.h"
+#include "software/geom/algorithms/contains.h"
+#include "software/simulated_tests/simulated_tactic_test_fixture.h"
+#include "software/simulated_tests/terminating_validation_functions/ball_kicked_validation.h"
+#include "software/simulated_tests/terminating_validation_functions/robot_state_validation.h"
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/test_util/test_util.h"
+#include "software/time/duration.h"
+#include "software/world/world.h"
+
+class SimulatedPasserTacticTest
+    : public virtual SimulatedTacticTestFixture,
```

Comment:
What is the significance of 'virtual'? Not all of the simulated tactic tests use it

Reply:
Ah good catch! This actually isn't needed at all

marking an inherited class [virtual basically prevents the diamond problem](https://en.wikipedia.org/wiki/Virtual_inheritance), in-order to get parameterized testing working w/ our simulated test fixture it _was_ required

But with `public ::testing::WithParamInterface` (which is what gtest recommends when using a custom test fixture), it seems to work without